frontend/ember-cli-build.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, {
+  ace: {
+    themes: ['cobalt','ambiance', 'chaos'],
+    modes: ['python'],
+    workers: ['python']
+  },
+  // Add more options here
+  });
+
+  app.import('bower_components/bootstrap/dist/js/bootstrap.min.js');
+  app.import('bower_components/bootstrap/dist/css/bootstrap.min.css');
+  app.import('bower_components/bootstrap/dist/css/bootstrap.min.css.map');
+
+  app.import('bower_components/bootstrap/dist/css/bootstrap-theme.min.css');
+  app.import('bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2', {
+    destDir: 'fonts'
+  });
+
+  app.import('vendor/css/narrow.css');
+  app.import('vendor/css/signup.css');
```

Comment:
If this is an app related css it, should go to the styles folder. Not in vendor folder.